### ISSUE TITLE
Had system scammed by Microsoft pop-up had to reinstalling everything requsting another pull of Last.fm scrobbler

### ISSUE THREAD

**Issue by knik23** [web-scrobbler/web-scrobbler#1665]
### Steps to reproduce (for bugs)

1.
2.
3.

### Steps for resolving the issue

- [ ] Reauthenticate on Last.fm

- [ ] Try to use unpacked extension from latest master

- [ ] Check other issues for duplicates

### Environment

* Version used:
* Browser name and version:

**Comment by Xalaxis** [comment]
I don't think anyone understands what this issue is supposed to mean.  Do you think you can clarify?

**Comment by bykidi** [comment]
probably just hacked account which sometimes spams to issues on various repositories.
happened to myself once, now i always use autogenerated passwords for every single account, and, ofc, 2FA

**Comment by knik23** [comment]
Are you people dense I was scammed with a Microsoft pop-up window(virus). When it happened wasn't thinking clearly(up for three nights) I let someone who claimed to be from Microsoft take control of my laptop, after I realized it a scam call FCC to report.  then wiped disk clean and installed everything again. I'm  asking for anything pull of David scrobbler. Can you understand that!!!

Kindly,

Angela

Get Outlook for Android<https://aka.ms/ghei36>

**Comment by knik23** [comment]
Ding Ding Ding you are correct. Read reply if you choose to.

Get Outlook for Android<https://aka.ms/ghei36>

**Comment by Xalaxis** [comment]
>I'm  asking for anything pull of David scrobbler. Can you understand that!!!

No.  Are you asking how to reinstall the extension perhaps?

**Comment by knik23** [comment]
Yes how is this so difficult. Did you read what happened bykidi was correct. I did download the extension that was available for chrome Last.fm on Monday it's David's and songs seem to be scrobbleing it that the lastest version.

Get Outlook for Android<https://aka.ms/ghei36>

**Comment by knik23** [comment]
david-sabata/web-scrobbler the guy that wrote addon for last.fm    https://add0n.com/lastfm-scrobbler.html   this is where I originally requested a pull from. I went to link above and it fixed my problem.



1.    Where I can find the change-log of new versions?

The change-log is available at https://github.com/david-sabata/web-scrobbler/releases.

2.    Some musics on Bandcamp cannot be scrobbled. Is it possible to fix this?

Looks like the music is hosted on a custom domain and we cannot detect whether it’s Bandcamp or not.

As a workaround, configure the extension to work with custom domains as follows:
1.    Open the extension setting page and expand the “Options” section,
2.    Find the Bandcamp connector in the list of supported services,
3.    Click on the gear icon near the connector label,
4.    Add URL pattern that matches the Bandcamp custom domain URL and apply the changes.

You can learn more about the custom URL patterns on github.com/david-sabata/…/Custom-URL-patterns<https://github.com/david-sabata/web-scrobbler/wiki/Custom-URL-patterns>

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
